common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -2,7 +2,18 @@
 # YOU CAN NOT INTERACT WITH THEM
 is_untouchable_trigger = {
 	hidden_tooltip = {
-		trait = sleeping_trait
+		OR = {
+			trait = sleeping_trait
+			AND = {
+				top_liege = { primary_title = { title = d_ulduar } } 
+				NOR = {
+					offmap_yoggsaron = { has_status = old_god_dead }
+					offmap_yoggsaron = { has_status = old_god_partly_free }
+					offmap_yoggsaron = { has_status = old_god_free }
+					offmap_yoggsaron = { has_status = old_god_full_power }
+				}
```

Comment:
So, can't you just use old_god_imprisoned OR old_god_partly_imprisoned, instead of 4 things in NOR?

Reply:
> So, can't you just use old_god_imprisoned OR old_god_partly_imprisoned, instead of 4 things in NOR?
i deleted condition and will add it back in branch that will allow that to happen

Reply:
fixed